components/webgpu/dom_messages.rs

```diff
@@ -255,6 +258,7 @@ pub enum WebGPURequest {
         device_id: id::DeviceId,
         pipeline_id: PipelineId,
     },
+    // COmpute pass
```

Comment:
'///'

Reply:
This isn't meant as a doccomment on BeginComputePass, but as a marker for section (next few enum variants are for compute pass).